src/helpers.js

```diff
@@ -112,7 +112,7 @@ function examplesForResource(resource) {
 
 function curlExamplesForAction(action, resource_examples) {
   var action_examples = _.filter(resource_examples, function(example) {
-    var regex = action.path.replace("%s", "\.+").replace(/\//, "\\/");
+    var regex = action.path.replace(/%s/g, "\\.+").replace(/\//g, "\\/");
```

Comment:
This doesnt work because the result is not a full regex object. It either needs to be  string (and maybe rename the var to reflect that) or it needs to be a full regex object so that str.match() does not try to convert it to one.